### ISSUE TITLE
flash light is not working in some devices

### ISSUE THREAD

**Issue by gagankumar007** [yuriy-budiyev/code-scanner#194]
There is a bug in the code-scanner library preventing the flashlight from turning on, on some devices.

**Comment by turuta7** [comment]
yes i have this problem in my device

**Comment by kimalbert92** [comment]
any solution ??

**Comment by grub39** [comment]
Exactly the same here on the Oneplus Nord 2 5G with Android 13. The flashlight simply cannot be activated :/

**Comment by PhalugunaRao** [comment]
any fix for this any one found solution?

**Comment by Linkan1992** [comment]
yes, flash is not working on VIVO Y33s and other same brand device.
